app/components/panel.tsx

```diff
@@ -39,88 +43,109 @@ export default function Panel({
   };
 
   return (
-    <div className="mb-auto bg-white rounded-md shadow-md lg:w-60 dark:shadow-gray-950/30 dark:bg-white/10">
+    <div>
+      <button
+        className="flex items-center w-full py-2 space-x-2 text-sm font-medium text-center lg:hidden "
+        onClick={() => setPanelVisibility(!isPanelVisible)}
+      >
+        <FunnelIcon className="w-4" />{" "}
+        <div className="text-sm">Filter Authors</div>{" "}
```

Comment:
No need to hold up the merge, but we should be using CSS to control the spacing here, not `{" "}` on the lines above.